### PR TITLE
Reset done dpi

### PR DESCRIPTION
**Pull request by mrutt92**
Changes to test bench code to accommodate CUDA Lite updates for BigBlade.

### CODE REVIEW

File: software/spmd/bsg_cuda_lite_runtime/dma/Makefile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#########################################################
+# Network Configutaion
+# If not configured, Will use default Values
+	bsg_global_X ?= $(bsg_tiles_X)
+	bsg_global_Y ?= $(bsg_tiles_Y)+1
```
Comment (drichmond) [inline software/spmd/bsg_cuda_lite_runtime/dma/Makefile:5]: Did this sneak in?

Comment (mrutt92) [reply software/spmd/bsg_cuda_lite_runtime/dma/Makefile:5]: The file? This is a simple test that I added.

Comment (drichmond) [reply software/spmd/bsg_cuda_lite_runtime/dma/Makefile:5]: One PR one feature...

Comment (mrutt92) [reply software/spmd/bsg_cuda_lite_runtime/dma/Makefile:5]: I vote that we be agile in this circumstance.

**Review by drichmond — APPROVED** [review]
💴

**Comment by mrutt92** [comment]
can we merge?

**Comment by mrutt92** [comment]
oh I guess I have perms

**Pull request merged**